### PR TITLE
Add pybroom recipe

### PR DESCRIPTION
**Pull request by tritemio**
First try at adding a recipe for pybroom.

### CODE REVIEW

**Comment by tritemio** [comment]
pybroom is a simple single-file pure-python 3+ package. After a few attempts builds are still failing. Any help is appreciated.

File: recipes/pybroom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python >=3.4
```
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: Please drop the version constraint.

File: recipes/pybroom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py27]
```
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: Please change `py27` to `py2k`.

File: recipes/pybroom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - pandas
+    - lmfit
+
+test:
+  imports:
+    - pybroom
+
+about:
+  home: http://pybroom.readthedocs.io/
+  license: MIT
```
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: We should be including the license file like so. However, that will require PR ( https://github.com/tritemio/pybroom/pull/4 ) to be merged and used to generate a release. Let's hold on that until we are confident we can get this to build right.

```yaml
  license_file: LICENSE.txt
```

**Comment by jakirkham** [comment]
Seems AppVeyor isn't starting. Opened this issue ( https://github.com/conda-forge/status/issues/9 ) to track it.

File: recipes/pybroom/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: Alright things seem to be building. Could we please do something about this license file?

**Comment by jakirkham** [comment]
Unrelated, but we have an [`lmfit` feedstock]( https://github.com/conda-forge/lmfit-feedstock ) if you would be interested in helping maintain that. It contains our recipe for `lmfit`.

File: recipes/pybroom/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "pybroom" %}
+{% set version = "0.2rc1" %}
```
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: We prefer to have a formal release. Is there a non-`rc` release we can use?

**Comment by tritemio** [comment]
@jakirkham, thanks!

I will soon release pybroom 0.2, the conda package can be based on that version (that will also have the license file).

And yes, I can help with the lmfit recipe if there is the need.

**Comment by tritemio** [comment]
@jakirkham I release pybroom 0.2 and updated the recipe, thanks!

File: recipes/pybroom/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: Done.

File: recipes/pybroom/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+{% set version = "0.2" %}
+{% set sha256 = "45761b81e64556a8a649f4561cdc2f102a269aff114e23dc93081762724cd123" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/pybroom/meta.yaml]: No need to template `name`, but if you are rolling out a bunch of personal packages like this I could see the value.

**Comment by jakirkham** [comment]
Thanks @tritemio.

**Comment by jakirkham** [comment]
Adding you as a maintainer of `lmfit` in PR ( https://github.com/conda-forge/lmfit-feedstock/pull/6 ).

**Pull request merged**